lib/AppInfo/Application.php

```diff
@@ -55,6 +59,12 @@ class Application extends App implements IBootstrap {
 
 	public function __construct() {
 		parent::__construct(self::APP_ID);
+
+        /* @var IEventDispatcher $eventDispatcher */
+        $dispatcher = $this->getContainer()->query(IEventDispatcher::class);
+
+        // Add photos button to files app
+        $dispatcher->addServiceListener(LoadAdditionalScriptsEvent::class, FilesListLoadingListener::class);
```

Comment:
Please use the `register` context instead. This is how we simplified the init since nextcloud 20 :)
https://github.com/nextcloud/server/blob/3e1916080af3ec2bc2fb7fd64f127cbd0c9b8a76/apps/comments/lib/AppInfo/Application.php#L65-L68

Reply:
Thanks, done :)